Classes/Information/Normalization/UseNormalizer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace FriendsOfTYPO3\Kickstarter\Information\Normalization;
+
+use Attribute;
+use FriendsOfTYPO3\Kickstarter\Command\Input\Normalizer\NormalizerInterface;
+
+#[Attribute(Attribute::TARGET_PROPERTY)]
```

Comment:
I have no problem with property. But currently you use it for string types properties:
```
#[UseNormalizer(EventClassNameNormalizer::class)]
private ?string $eventClassName = null;
```